src/xml/EllipseXML.java

```diff
@@ -0,0 +1,41 @@
+package xml;
+
+import view.PencilState;
+import model.specific_path.Ellipse;
+
+public class EllipseXML extends ShapeXML {
+	
```

Comment:
That wasalso what I meant. Creating ShapesXML in a way that the user doesn't have to it himself